zulip_bots/zulip_bots/run.py

```diff
@@ -3,12 +3,33 @@
 from __future__ import absolute_import
 
 import logging
-import optparse
+import argparse
 import sys
+import os
 from types import ModuleType
 from importlib import import_module
+from os.path import basename, splitext
+
+import six
 
 from zulip_bots.lib import run_message_handler_for_bot
+from zulip_bots.provision import provision_bot
+
+
+def import_module_from_source(path, name=None):
+    if name is None:
+        name = splitext(basename(path))[0]
+
+    if six.PY2:
```

Comment:
I think we should plan to support both.  Very few organizations do Python 3 yet, and this is classically the type of code you want to be able to run anywhere.